test/integration/targets/remote_temp/playbook.yml

```diff
@@ -0,0 +1,15 @@
+- name: Test temp dir on de escalation
+  hosts: testhost
+  gather_facts: false
+  tasks:
+    - block:
+        - blockinfile:
+              create: yes
+              block: |
+                export foo=bar
+              marker: "# {mark} Here there be a marker"
+              dest: /tmp/testing.txt
+              mode: 0644
+        - file: path=/tmp/testing.txt state=absent
+      become_user: nobody
```

Comment:
nobody often has a home dir that can't be written to.  Perhaps the playbook has to create a user for this test?